spark/src/main/kotlin/com/adevinta/spark/components/dialog/ModalScaffold.kt

```diff
@@ -137,6 +137,7 @@ public fun ModalScaffold(
                 )
     val activityWindow = getActivityWindow()
 
+    @Suppress("DEPRECATION")
```

Comment:
@soulcramer do we need to do something about these APIs being used while deprecated? Are they still working as expected?

Reply:
I checked and without them it works as expected with edge-to-edge and without 🤷

Reply:
On all API levels you mean? (pre&post forced edge-to-edge)

Reply:
yes I tested it on my emulator on 31 but I did not tested it without gestures, I'll try with that

Reply:
I changed the Modal Scaffold API here https://github.com/adevinta/spark-android/pull/1393 to optin the edge to edge support, on early version it didn't work as expected with the navigation buttons where the bottom bar would draw behind it instead of on top of it